v1/pipeline/stages/ci/failure.yaml

```diff
@@ -0,0 +1,98 @@
+title: Failure
+type: object
+description: Failure defines a failure strategy.
+properties:
```

Comment:
We should point to the common failure strategy or values which are common for all modules, we can move the common ones into common folder so that all services can adopt.

@shalini-agr can help do that